src/containers/Guidelines.jsx

```diff
@@ -4,23 +4,49 @@ import Layout from '../components/Layout.jsx'
 import s from './Guidelines.css';
 
 class Guidelines extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isVisible: false
+    };
+  }
+
   render() {
-    return(
+    let buttonBody;
+    if (this.state.isVisible) {
+      buttonBody = this.props.body;
```

Comment:
where do we get the `body` props from ?

Reply:
Oups ! 
J'avais commencé à essayer de bidouiller cela il y a des mois, et oublié...
The first two commits have to be forgotten, just cherry pick the third, or i delete my branch, refork and just modify the kata.js...

Reply:
yes I think it is simpler if you make a branch from master and just add the kata to the kata list